### PR TITLE
Add new profile for SQL Server

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
I'll merge these commits with existing commits.

File: presto-product-tests/conf/docker/singlenode-sqlserver/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+version: '2'
```
Comment (ArturGajowy) [inline presto-product-tests/conf/docker/singlenode-sqlserver/docker-compose.yml:1]: move the sqlserver container from standard.yml to here

File: presto-product-tests/bin/run_on_docker.sh [hunk 2]
```diff
@@ -194,7 +194,11 @@ fi
 trap terminate INT TERM EXIT
 
 # start external services
-EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra sqlserver"
+if [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
+  EXTERNAL_SERVICES="hadoop-master cassandra sqlserver"
```
Comment (ArturGajowy) [inline presto-product-tests/bin/run_on_docker.sh]: add a comment why cassandra is needed here

**Review by ArturGajowy — CHANGES_REQUESTED** [review]

File: presto-product-tests/bin/run_on_docker.sh (hunk 2 above)
Comment (ArturGajowy) [reply presto-product-tests/bin/run_on_docker.sh]: the comment should link to a tempto issue that - once solved - will allow us to not have cassandra here. If there's not such an issue, create it.

File: presto-product-tests/conf/docker/singlenode-sqlserver/docker-compose.yml (hunk 1 above)
Comment (sanjay990) [reply presto-product-tests/conf/docker/singlenode-sqlserver/docker-compose.yml:1]: @ArturGajowy https://github.com/Teradata/presto/pull/504/commits/ab6a78fd88af3561cbe091ad77e69b7cddcc716a

**Comment by ArturGajowy** [comment]
fixup looks good!

**Review by ArturGajowy — APPROVED** [review]

**Comment by sanjay990** [comment]
Merged

**Pull request closed without merging**